Source/JavaScriptCore/wasm/WasmOMGIRGenerator32_64.cpp

```diff
@@ -98,6 +98,7 @@ using ConstIntPtrValue = Const32Value;
 using ConstIntPtrValue = Const64Value;
 #endif
 
+namespace {
```

Comment:
Why is this extra namespace needed here?

Reply:
This is there to keep this file as close as possible to the 64bit version. In particular, the [reverting changes on WasmOMGIRGenerator.cpp also re-add this extra namespace](https://github.com/WebKit/WebKit/pull/36068/files#diff-e0cc9beb26b9d2cbf93a0dbc593d1ccfbd186ade0356b40ff01a87a85e5c0486R98).